src/coreclr/tools/aot/ILCompiler/ILCompiler.props

```diff
@@ -48,8 +48,7 @@
 
     <!-- CoreDisTools are used in debugging visualizers. -->
     <IncludeCoreDisTools Condition="'$(Configuration)' != 'Release' and '$(CrossHostArch)' == ''">true</IncludeCoreDisTools>
-    <!-- source-build doesn't use ObjWriter for the ILCompiler.  the end-user will end up pulling Microsoft-built bits for NativeAOT anyway.  -->
-    <IncludeObjWriter Condition="'$(DotNetBuildFromSource)' != 'true'">true</IncludeObjWriter>
+    <IncludeObjWriter>true</IncludeObjWriter>
```

Comment:
Could you do one more level of constant propagation on this and also delete the property?